### PR TITLE
New font manager layout

### PR DESCRIPTION
**Pull request by debbieellis7**
## Description

New react font-manager layout component & removal of underscore and backbone JS depenencies. This branch is created under 'gf-2.5'.

Part of issue #738 

## Checklist:
- [x] I've tested the code.
- [x] My code is easy to read, follow, and understand
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation / docblocks.

### CODE REVIEW

File: src/assets/js/admin/settings/common/dynamicTemplateFields/doMergetags.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import $ from 'jquery'
  */
 export function doMergetags () {
   if (window.gfMergeTags && typeof form !== 'undefined' && $('#gfpdf-fieldset-gfpdf_form_settings_template .merge-tag-support').length >= 0) {
-
```
Comment (jakejackson1) [inline src/assets/js/admin/settings/common/dynamicTemplateFields/doMergetags.js:10]: Irrelivent for this PR

File: src/assets/js/admin/settings/pdf/doFormSettingsEditPage.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export function doFormSettingsEditPage () {
 
   /* Hide Template section, if empty */
   const $templateSection = $('#gfpdf-fieldset-gfpdf_form_settings_template')
-  if($templateSection.find('.gform-settings-panel__content').children().length === 0) {
+  if ($templateSection.find('.gform-settings-panel__content').children().length === 0) {
```
Comment (jakejackson1) [inline src/assets/js/admin/settings/pdf/doFormSettingsEditPage.js:33]: Irrelivent for this PR

File: src/assets/js/react/utilities/CloseDialog.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router-dom'
+
+/**
+ * Renders our close dialog element
+ *
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       4.1
```
Comment (jakejackson1) [inline src/assets/js/react/utilities/CloseDialog.js:11]: Should be 6.0

File: src/assets/scss/Component/_color-picker.scss [hunk 4]
```diff
@@ -1,30 +1,10 @@
-@use '../Mixins/button';
-
-#tab_pdf, #tab_PDF {
+#tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_color-picker.scss:1]: This file shouldn't have changed.

File: src/assets/scss/Component/_gf-settings-containers.scss [hunk 5]
```diff
@@ -1,5 +1,4 @@
-
-#tab_pdf, #tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_gf-settings-containers.scss:2]: This line shouldn't have changed

File: src/assets/scss/Component/_font-manager.scss [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+@mixin installed-fonts {
+  width: 40%;
+}
+
+@mixin font-color-regular {
+  color: #434563;
+}
+
+#tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_font-manager.scss:16]: Needs `#tab_pdf` as well

File: src/assets/scss/Component/_license.scss [hunk 7]
```diff
@@ -1,5 +1,5 @@
-#tab_pdf, #tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_license.scss:1]: This line shouldn't have changed

File: src/assets/scss/Component/_multicheck.scss [hunk 8]
```diff
@@ -1,4 +1,4 @@
-#tab_pdf, #tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_multicheck.scss:1]: This line shouldn't have changed

File: src/assets/scss/Component/_rich_text.scss [hunk 9]
```diff
@@ -1,6 +1,5 @@
-@use '../Mixins/button';
```
Comment (jakejackson1) [inline src/assets/scss/Component/_rich_text.scss:1]: This file shouldn't have changed

**Review by jakejackson1 — CHANGES_REQUESTED** [review]

File: src/assets/scss/Component/_rtl.scss [hunk 10]
```diff
@@ -1,11 +1,7 @@
-
```
Comment (jakejackson1) [inline src/assets/scss/Component/_rtl.scss:1]: This file shouldn't have changed

File: src/assets/scss/Component/_spinner.scss [hunk 11]
```diff
@@ -1,28 +1,20 @@
-#tab_pdf, #tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_spinner.scss:1]: This file shouldn't have changed

File: src/assets/scss/Component/_template-manager.scss [hunk 12]
```diff
@@ -1,50 +1,28 @@
-@use '../Variables/colors';
-@use '../Mixins/button';
-
-#tab_pdf, #tab_PDF {
+@use '../base';
```
Comment (jakejackson1) [inline src/assets/scss/Component/_template-manager.scss:1]: Revert everything but your variable migration

File: src/assets/js/admin/settings/common/dynamicTemplateFields/doMergetags.js (hunk 1 above)
Comment (debbieellis7) [reply src/assets/js/admin/settings/common/dynamicTemplateFields/doMergetags.js:10]: JS lint error

File: src/assets/scss/Component/_color-picker.scss [hunk 13]
```diff
@@ -2,7 +2,6 @@
 
 #tab_pdf, #tab_PDF {
```
Comment (jakejackson1) [inline src/assets/scss/Component/_color-picker.scss:3]: These changes can be removed altogether.

File: src/assets/scss/Component/_gf-settings-containers.scss [hunk 14]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (jakejackson1) [inline src/assets/scss/Component/_gf-settings-containers.scss:1]: These changes in this file can be removed all together

File: src/assets/scss/Component/_help.scss [hunk 15]
```diff
@@ -50,24 +47,18 @@
...
 
-          h3 {
-            margin-bottom: 0;
-
-            .hndle {
-              padding-bottom: 12px;
-
-              .spinner {
+          .hndle {
+            padding: 12px;
```
Comment (jakejackson1) [inline src/assets/scss/Component/_help.scss:53]: Why the move from `padding-bottom` to `padding: 12px`?

File: src/assets/scss/Component/_license.scss [hunk 16]
```diff
@@ -2,4 +2,4 @@
   .gfpdf-deactivate-license {
     margin-top: 0.5rem;
   }
-}
```
Comment (jakejackson1) [inline src/assets/scss/Component/_license.scss:5]: These changes in this file can be removed all together

File: src/assets/scss/Component/_multicheck.scss [hunk 17]
```diff
@@ -7,4 +7,4 @@
     float: left;
     width: 33%;
   }
-}
```
Comment (jakejackson1) [inline src/assets/scss/Component/_multicheck.scss:10]: These changes in this file can be removed all together

File: src/assets/scss/Component/_number.scss [hunk 18]
```diff
@@ -2,4 +2,4 @@
   input[type="number"].small-text {
     width: 5rem;
   }
-}
```
Comment (jakejackson1) [inline src/assets/scss/Component/_number.scss:5]: These changes in this file can be removed all together

File: src/assets/scss/Component/_rich_text.scss [hunk 19]
```diff
@@ -1,6 +1,7 @@
 @use '../Mixins/button';
 
 #tab_pdf, #tab_PDF {
+
```
Comment (jakejackson1) [inline src/assets/scss/Component/_rich_text.scss:4]: These changes in this file can be removed all together

File: src/assets/scss/Component/_spinner.scss [hunk 20]
```diff
@@ -1,28 +1,20 @@
 #tab_pdf, #tab_PDF {
-
   .gfpdf-spinner {
     padding: 0 5px;
     vertical-align: middle;
-    width: 27px;
+    width: 21px;
```
Comment (jakejackson1) [inline src/assets/scss/Component/_spinner.scss:5]: Revert back to 27px please. We're going for a bigger spinner as standard.

File: src/assets/scss/Component/_submit.scss [hunk 21]
```diff
@@ -9,4 +9,4 @@
       font-style: italic;
     }
   }
-}
```
Comment (jakejackson1) [inline src/assets/scss/Component/_submit.scss:12]: These changes in this file can be removed all together

File: src/assets/scss/Component/_upload.scss [hunk 22]
```diff
@@ -16,4 +16,4 @@
       margin-left: 0;
     }
   }
-}
```
Comment (jakejackson1) [inline src/assets/scss/Component/_upload.scss:19]: These changes in this file can be removed all together

File: src/assets/scss/Pages/_entry-details.scss [hunk 23]
```diff
@@ -1,12 +1,7 @@
-/*
- * Form -> Entries -> Individual Entry
```
Comment (jakejackson1) [inline src/assets/scss/Pages/_entry-details.scss:2]: These changes in this file can be removed all together

File: src/assets/scss/Pages/_pdf-list.scss [hunk 24]
```diff
@@ -1,14 +1,8 @@
-/*
- * Styles for the Forms -> PDF page
- */
```
Comment (jakejackson1) [inline src/assets/scss/Pages/_pdf-list.scss:3]: These changes in this file can be removed all together

File: src/assets/scss/_rtl.scss [hunk 25]
```diff
@@ -1,11 +1,7 @@
-
```
Comment (jakejackson1) [inline src/assets/scss/_rtl.scss:1]: These changes in this file can be removed all together

**Review by jakejackson1 — CHANGES_REQUESTED** [review]

File: src/assets/scss/Component/_help.scss (hunk 15 above)
Comment (debbieellis7) [reply src/assets/scss/Component/_help.scss:53]: This is for the size of the title back which contain the text 'Gravity PDF Documentation'. It looks very small.

Comment (debbieellis7) [reply src/assets/scss/Component/_help.scss]: reverted

**Pull request merged**